_data/blog.yml

```diff
@@ -4,34 +4,36 @@
 # - Site has also loaded: https://itenium.be/vendor/flat-icon/flaticon.html
 
 categories:
-  javascript: icon-javascript
-  dotnet: icon-csharp
-  productivity: fa fa-superpowers
-  dev-setup: fa fa-cogs
-  design: fa fa-object-group
+    javascript: icon-javascript
+    dotnet: icon-csharp
+    productivity: fa fa-superpowers
+    dev-setup: fa fa-cogs
+    design: fa fa-object-group
 
 # fa fa-battery-full / fa fa-battery-three-quarters
 # fa fa-bolt / fa fa-cogs / fa fa-line-chart
 
-
 tags:
-  powershell: icon-shell
-  net: icon-csharp
-  javascript: icon-javascript
-  autohotkey: fa fa-key
-  git: icon-git
-  windows: fa fa-windows
-  excel: fa fa-file-excel-o
-  testing: fa fa-diamond
-  tutorial: fa fa-cogs
-  "cheat-sheet": fa fa-bookmark
-  angular: icon-angular
-  sql: icon-mssql
-  "war-story": fa fa-lightbulb-o
-  product: fa fa-money
-  debugging: fa fa-bug
-  meta: fa fa-tag
-  regex: icon-script
-  tech-talk: fa fa-certificate
-  pragmatic-tips: fa fa-trophy
-  fun: fa fa-gamepad
+    powershell: icon-shell
+    net: icon-csharp
+    javascript: icon-javascript
+    autohotkey: fa fa-key
+    git: icon-git
+    windows: fa fa-windows
+    excel: fa fa-file-excel-o
+    testing: fa fa-diamond
+    tutorial: fa fa-cogs
+    "cheat-sheet": fa fa-bookmark
+    angular: icon-angular
+    sql: icon-mssql
+    "war-story": fa fa-lightbulb-o
+    product: fa fa-money
+    debugging: fa fa-bug
+    meta: fa fa-tag
+    regex: icon-script
+    tech-talk: fa fa-certificate
+    pragmatic-tips: fa fa-trophy
+    fun: fa fa-gamepad
+    maui: icon-csharp
+    mobile: fa fa-mobile
+    desktop-app: fa fa-desktop
```

Comment:
Hey Tim,

Wat is er hier precies veranderd?

De indenting zou reeds moeten goed staan -- ie in sync met de .editorconfig
Als dat niet het geval is, dan wil ik deze PR wel mergen, maar als de nieuwe layout foutief is voor de .editorconfig, kun je dit dan nog aanpassen?